examples/tutorial_examples/add_custom_bundle.py

```diff
@@ -0,0 +1,233 @@
+"""
+=================================================
+Adding new bundles into pyAFQ (SLF 1/2/3 Example)
+=================================================
+
+pyAFQ is designed to be customizable and extensible. This example shows how you
+can customize it to define a new bundle based on a definition of waypoint and
+endpoint ROIs of your design.
+
+In this case, we add sub-bundles of the superior longitudinal fasciculus,
+based on work by Rami et al [1]_.
+
+We start by importing some of the components that we need for this example and
+fixing the random seed for reproducibility
+"""
+
+import os.path as op
+import plotly
+import numpy as np
+import shutil
+
+from AFQ.api.group import GroupAFQ
+import AFQ.api.bundle_dict as abd
+import AFQ.data.fetch as afd
+from AFQ.definitions.image import ImageFile, RoiImage
+import wget
+import os
+np.random.seed(1234)
+
+
+#############################################################################
+# Get dMRI data
+# ---------------
+# We will analyze eight subject from the Healthy Brain Network Processed Open
+# Diffusion Derivatives dataset (HBN-POD2) [2]_, [3]_. We'll use a fetcher to
+# get preprocessed dMRI data for eight of the >2,000 subjects in that study. The
+# data gets organized into a BIDS-compatible format in the `~/AFQ_data/HBN`
+# folder. These 12 subjects have very high quality data.
+# The fether returns this directory as study_dir:
+
+_, study_dir = afd.fetch_hbn_preproc(['NDARKP893TWU',
+'NDAREP505XAD',
+'NDARKT540ZW0',
+'NDARAG340ERT',
+'NDAREM757NBG',
+'NDARLL894HC3',
+'NDARFY525TL2',
+'NDARKV461KGZ',
+'NDARUC851WHU',
+'NDARMJ333WJM',
+'NDARJG687YYX',
+'NDARJA157YB3',
+])
```

Comment:
Just a white-space suggestion, to improve readability (and pep8 compliance)
```suggestion
_, study_dir = afd.fetch_hbn_preproc([
    'NDARKP893TWU',
    'NDAREP505XAD',
    'NDARKT540ZW0',
    'NDARAG340ERT',
    'NDAREM757NBG',
    'NDARLL894HC3',
    'NDARFY525TL2',
    'NDARKV461KGZ',
    'NDARUC851WHU',
    'NDARMJ333WJM',
    'NDARJG687YYX',
    'NDARJA157YB3',
])
```